modules/core/04-channel/types/errors.go

```diff
@@ -58,4 +58,6 @@ var (
 	ErrTimeoutElapsed                  = errorsmod.Register(SubModuleName, 40, "timeout elapsed")
 	ErrPruningSequenceStartNotFound    = errorsmod.Register(SubModuleName, 41, "pruning sequence start not found")
 	ErrRecvStartSequenceNotFound       = errorsmod.Register(SubModuleName, 42, "recv start sequence not found")
+	ErrInvalidCommitment               = errorsmod.Register(SubModuleName, 43, "invalid commitment")
+	ErrInvalidSend                     = errorsmod.Register(SubModuleName, 44, "invalid send")
```

Comment:
I think these new errors help in making the error messages more structured and reusable. They also align with the errorsmod convention we are following.

Reply:
I think ErrInvalidSequence would be better than ErrInvalidSend as it can be used for next channel sequence as well and is more understandable than NextSend.

ErrInvalidCommitment seems reasonable.

Reply:
thanks u